python/pipeline/pupil.py

```diff
@@ -596,27 +596,75 @@ class Tracking(dj.Computed):
     tracking_ts=CURRENT_TIMESTAMP   : timestamp  # automatic
     """
 
+    class ManualTrackingParameter(dj.Part):
```

Comment:
I decided to separate out because the ManualTracking already has `blob` in it, which in theory takes longer than any other data types. Idk when and why we would ever need parameter info separately, but if someone wanna see the parameters, i don't want the `blob` to drag down the performance of data transfer. So I separated out

Reply:
this tables are gonna have a lot of rows if they have one per frame, I wonder whether that is ok, I feel like that will be way to slow maybe we have one entry per scan and have an external blob with all th results per frame stacked together

Reply:
parameters, in most cases, are the same from frame to frame. But Talish said that there are cases she had to adjust parameters while tracking, hence there is no guarnatee that every frame has the same parameters. 

Imo, this is OK since from now on, most of them are gonna be automatically tracked. The only case we will ever need to access the parameters table is when we need to look at the _old_ scans that were previously scanned manually...

Reply:
so the automatic method won't have one entry per frame, right?

Reply:
automatic method won't and doesnt even have parameters table. Parameters table only applies to manual